src/chapters/intro-stats-data-modeling/intro-stats-data-modeling.rst

```diff
@@ -1,2 +1,141 @@
 Introduction to Statistics and Data Modeling
-============================================
\ No newline at end of file
+============================================
+
+Readings
+--------
+
+Read the following articles, and watch the videos.
+
+
+`Descriptive vs. Inferential Statistics <https://www.thoughtco.com/differences-in-descriptive-and-inferential-statistics-3126224>`__.
+
+* Don't be afraid to explore the links in the article to get a more in depth understanding of the material.
+
+`Understanding Descriptive Statistics | by Sarang Narkhede <https://towardsdatascience.com/understanding-descriptive-statistics-c9c2b0641291>`__.
+
+`How to choose the best chart or graph for your data <https://cloud.google.com/blog/products/data-analytics/different-types-graphs-charts-uses>`__.
+
+`Python Statistics Fundamentals: How to Describe Your Data – Real Python <https://realpython.com/python-statistics/#visualizing-data>`__.
```

Comment:
Just want to check that having this link directly to the smaller section on Visualizing Data was intentional.